src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -172,6 +171,12 @@ public void updateTimeLineForRepartition(final EventType eventType, final int pa
             }
             timelineDbRepository.updateTimelime(timeline);
         }
+
+        try {
+            eventTypeCache.updated(eventType.getName());
+        } catch (Exception e) {
+            throw new NakadiBaseException(e.getMessage(), e);
```

Comment:
Is there a better exception we could throw?

Reply:
I do not see a reason for that, at the end it is converted to internal server error in the calling method